api-module-library/slack/api.js

```diff
@@ -120,7 +120,9 @@ class Api extends OAuth2Requester {
         const parsedResponse = await this.parsedBody(response);
         const { status } = response;
         const { ok, error } = parsedResponse;
-        console.log(parsedResponse);
```

Comment:
I just saw another PR removes this on the main branch. https://github.com/friggframework/frigg/pull/298

I think that probably makes more sense for this particular log line.

Reply:
Updated the PR to just remove that line instead (matching the PR on `main`)